src/components/Steps/Steps.tsx

```diff
@@ -30,10 +30,16 @@ interface StepsProperties {
 }
 
 export const Steps = (props: StepsProperties) => {
-    const children = (Array.isArray(props.children) ? props.children : (props.children ? [props.children] : [])).concat([]);
+    const children = (
+        Array.isArray(props.children)
+            ? props.children
+            : props.children
+            ? [props.children]
+            : []
+    ).concat([]);
```

Comment:
The removal of the parens here is unfortunate, but good elsewhere so probably net positive I suppose